File: contestOfKotlin/src/test/kotlin/challenges/Challenge4Test.kt
package challenges.challenge4

import org.example.challenges.challenge4.*
import kotlin.test.*

class Challange4TestCase {
    @Test fun testIsPrime() {
        assertTrue(isPrime(2))
        assertTrue(isPrime(3))
        assertTrue(isPrime(5))
        assertTrue(isPrime(7))
        assertTrue(isPrime(23))
        assertTrue(isPrime(31))
        assertTrue(isPrime(73))

        assertFalse(isPrime(1))
        assertFalse(isPrime(4))
        assertFalse(isPrime(9))
        assertFalse(isPrime(15))
        assertFalse(isPrime(21))
        assertFalse(isPrime(27))
        assertFalse(isPrime(30))
    }
}

File: contest_of_dart/test/challenges/challenge4_test.dart
import 'package:test/test.dart';
import '../../bin/challenges/challenge4.dart';

void main() {
  test("Challenge 4: Is prime number", () {
    expect(
        Challenge4.isPrime(2),
        equals(true)
    );
    expect(
        Challenge4.isPrime(3),
        equals(true)
    );
    expect(
        Challenge4.isPrime(5),
        equals(true)
    );
    expect(
        Challenge4.isPrime(7),
        equals(true)
    );
    expect(
        Challenge4.isPrime(23),
        equals(true)
    );
    expect(
        Challenge4.isPrime(31),
        equals(true)
    );
    expect(
        Challenge4.isPrime(73),
        equals(true)
    );

    expect(
        Challenge4.isPrime(1),
        equals(false)
    );
    expect(
        Challenge4.isPrime(4),
        equals(false)
    );
    expect(
        Challenge4.isPrime(9),
        equals(false)
    );
    expect(
        Challenge4.isPrime(15),
        equals(false)
    );
    expect(
        Challenge4.isPrime(21),
        equals(false)
    );
    expect(
        Challenge4.isPrime(27),
        equals(false)
    );
    expect(
        Challenge4.isPrime(30),
        equals(false)
    );
  });
}